### ISSUE TITLE
rc configuration file not detected in the current directory

### ISSUE THREAD

**Issue by jamesmortensen** [node-inspector/node-inspector#424]
According to the documentation, I can set a configuration file in the current directory to override default settings:

> a local .node-inspectorrc or the first found looking in ./ ../ ../../ ../../../ etc.

I created `.node-inspectorrc` and placed the following configuration inside:

``` javascript
{
  "web-port": 9080,
  "web-host": null,
  "debug-port": 5858,
  "save-live-edit": true,
  "no-preload": true,
  "hidden": [],
  "stack-trace-limit": 50
}
```

However, when running `node-debug app.js` the port still defaults to 8080.  It appears as if the debugger is not in fact checking inside `./` for the file.  Hope this helps and thank you for developing/maintaining this great tool.
